firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -15,13 +15,21 @@
 package com.google.firebase.installations;
 
 import android.text.TextUtils;
+
+import androidx.annotation.NonNull;
+
 import com.google.firebase.installations.local.PersistedInstallationEntry;
 import java.util.concurrent.TimeUnit;
 
+import com.google.re2j.Pattern;
```

Comment:
We have to make sure that this package `re2j` is available on all Android devices.
Can you please check with someone from Android team.